pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -950,12 +950,26 @@ func SetDefaults(cr *api.KieApp) {
 			retainAppliedPwds(&specApply.Objects.Servers[index], statusServer)
 		}
 		addWebhookPwds(specApply.Objects.Servers[index].Build)
+		setJvmDefault(specApply.Objects.Console.Jvm)
```

Comment:
since this doesn't affect the `specApply.Objects.Servers` object, it can be moved it out of the for loop it's currently in... maybe up to line 946?

Reply:
Ok